internal/cmd/local/command_broker_describe.go

```diff
@@ -0,0 +1,83 @@
+package local
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/broker"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+func (c *Command) newBrokerDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe [id]",
+		Short: "Describe a Confluent local broker.",
+		Long:  "Describe cluster-wide or per-broker configuration values.",
+		Args:  cobra.MaximumNArgs(1),
+		RunE:  c.describe,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Describe the "min.insync.replicas" configuration for broker 1.`,
+				Code: "confluent local broker describe 1 --config-name min.insync.replicas",
+			},
+			examples.Example{
+				Text: "Describe the non-default cluster-wide broker configuration values.",
+				Code: "confluent local broker describe --all",
+			},
+		),
+	}
+
+	cmd.Flags().Bool("all", false, "Get cluster-wide broker configurations (non-default values only).")
```

Comment:
As I mentioned above, I would like to delete this and replace this with `confluent local kafka cluster configuration list` to match the existing CP command.

Reply:
we have this flag in CP `confluent kafka broker describe --all` too. Are cluster configs and a cluster-wide broker config the same?